### ISSUE TITLE
compiler 작동 오류(?) 질문

### ISSUE THREAD

**Issue by hyokkkk** [snu-sf-class/swpp202101#32]
안녕하세요,

업로드해주신 compiler를 사용해 이것저것 실험하면서 .ll파일을 .s로 변환시켜 interpreter로 돌려보고 있었습니다. 그런데 어떤 .ll 파일은 .s 파일이 안 만들어지는 것을 발견하였는데요, .ll 파일 코드 중에 잘못된 부분이 있어서 그런 건가요? 제가 임의로 (아마도 잘못된 문법으로) 추가한 부분을 지우면 .s 파일이 생성되기는 하는데, 아무런 에러메시지도 뜨지 않아서 질문드립니다.

**Comment by lego0901** [comment]
혹시 오류가 나는 .ll 파일을 조교 이메일로 보내주실 수 있으신가요?

**Comment by hyokkkk** [comment]
답장 잘 받았습니다! 그러면 아무런 메시지 없이 .s파일이 생성되지 않으면 .ll코드에 문제가 있는 것으로 알면 될까요?

**Comment by lego0901** [comment]
네, 다만 필요하시다면 src/core/main.cpp 파일에서 오류가 생긴 경우 에러 메세지를 출력하도록 수정하시고 쓰셔도 됩니다.

- 송우성 드림

**Comment by hyokkkk** [comment]
넵 감사합니다!
